rasa_nlu/tokenizers/__init__.py

```diff
@@ -0,0 +1,19 @@
+def tokenizer_from_name(name, language):
+    from rasa_nlu.tokenizers.whitespace_tokenizer import WhitespaceTokenizer
+    from rasa_nlu.tokenizers.mitie_tokenizer import MitieTokenizer
+    from rasa_nlu.tokenizers.spacy_tokenizer import SpacyTokenizer
+
+    if name == MitieTokenizer.name:
+        return MitieTokenizer()
+    elif name == SpacyTokenizer.name:
+        import spacy
+        nlp = spacy.load(language, parser=False, entity=False)
```

Comment:
but if you're running the server with a spacy model, won't the SpacyTokenizer get instantiated at some point? and that would load a new `nlp` obj?